packages/raja/package.py

```diff
@@ -455,7 +455,7 @@ def cmake_args(self):
     @property
     def build_relpath(self):
         """Relative path to the cmake build subdirectory."""
-        return join_path("..", self.builder.build_dirname)
+        return join_path("..", self.build_dirname)
```

Comment:
Thanks @adrienbernede ! You are fixing my problems before I even hit them!

Reply:
This PR should go in soon BTW.